src/coreclr/tests/src/Directory.Build.targets

```diff
@@ -192,11 +193,12 @@
 
   <Target Name="CopyAllNativeProjectReferenceBinaries" DependsOnTargets="ResolveCmakeNativeProjectReference;ConsolidateNativeProjectReference" />
 
+  <!-- Condition="'$(_CopyNativeProjectBinaries)' == 'true' And '$(GenerateRunScript)' != 'false' And ('$(_WillCLRTestProjectBuild)' == 'true')" -->
   <!-- Build shell or command scripts whenever we copy native binaries -->
   <Import Project="CLRTest.Execute.targets" />
   <Target Name="CreateExecuteScript"
           BeforeTargets="Build;CopyAllNativeProjectReferenceBinaries"
-          Condition="'$(_CopyNativeProjectBinaries)' == 'true' And '$(GenerateRunScript)' != 'false' And ('$(_WillCLRTestProjectBuild)' == 'true')"
```

Comment:
I'm starting to think there's some problem with removal of this condition - I originally added that to the <code>_PropertiesToPass</code> and all non-Windows legs failed in weird ways. I think that what most probably happens is that the execution scripts end up getting emitted multiple times and the last version doesn't have all the target parameters properly set. I'm investigating that right now.

Reply:
I can maybe change the condition based one it being Windows (it will always be "Browser" for wasm), but that sounds like introducing a bug to counter-act another bug.

Reply:
So, looking through all the scripts, it looks like this gets hit when we copy the native binaries. It may be as simple as passsing "/p:GenerateRunScript=false" from build-test.sh when we do that... Trying that.

Reply:
Well, that didn't work.